internal/util/paramtable/component_param.go

```diff
@@ -748,21 +752,17 @@ func (p *queryNodeConfig) initOverloadedMemoryThresholdPercentage() {
 	p.OverloadedMemoryThresholdPercentage = float64(thresholdPercentage) / 100
 }
 
+func (p *queryNodeConfig) initLocalFileCacheLimit() {
+	p.LocalFileCacheLimit = p.Base.ParseUint64WithDefault("queryNode.cache.localFileCacheLimit", 2147483648)
```

Comment:
localFileCacheLimit should be far larger than memoryLimit.
memoryLimit -> 256MB
localFileCacheLimit -> 16GB